assets/scripts/util/street_analytics.js

```diff
@@ -61,8 +61,12 @@ const CAPACITIES = {
   'magic-carpet': { average: 2, potential: 3 }
 }
 
+const hasCapacityType = (type) => {
+  return type in CAPACITIES
+}
+
 export const getCapacity = (type) => {
-  return CAPACITIES[type] || UNDEFINED_CAPACITY
+  return hasCapacityType(type) ? { ...CAPACITIES[type], display: true } : UNDEFINED_CAPACITY
```

Comment:
See above comment - if that makes sense, it would avoid having to do a spread operator here to add `display: true` to the capacity definition.